src/Dfc.CourseDirectory.Web/Views/Apprenticeships/Complete/Index.cshtml

```diff
@@ -1,33 +1,37 @@
-﻿@using Microsoft.AspNetCore.Authorization;
-@using Microsoft.AspNetCore.Identity
-@using Dfc.CourseDirectory.Services.Models
+﻿@addTagHelper *, GovUk.Frontend.AspNetCore
 @model Dfc.CourseDirectory.Web.ViewModels.Apprenticeships.CompleteViewModel
 @{
-    ViewData["Title"] = Model.ApprenticeshipName + " has been added to the Course directory"; 
-    Layout = "_Layout_Your_Courses";
+    ViewData["Title"] = Model.ApprenticeshipName + " has been added to the Course directory";
+    Layout = "_V2LayoutProviderContext";
 }
 
-<div class="govuk-panel govuk-panel--confirmation">
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <govuk-panel>
+            <govuk-panel-title>Apprenticeship published</govuk-panel-title>
 
-    <div class="govuk-panel__body">
-        <h1 class="govuk-panel__title">
-            Apprenticeship published
-        </h1>
+            @Model.ApprenticeshipName
```

Comment:
Should this be enclosed in a `govuk-panel-body` helper tag? It's missing the `govuk-panel__body` style

The generated HTML should look like this:
```
<div class="govuk-panel govuk-panel--confirmation">
    <h1 class="govuk-panel__title">
        Apprenticeship published
    </h1>
    <div class="govuk-panel__body">
		[apprenticeship name]
	</div>
</div>
```

Reply:
the tag helper does it automagically

![Selection_20210204-01-d331ed331ed331e](https://user-images.githubusercontent.com/19378/106899788-af581600-66ed-11eb-8089-484346463bc3.png)

Reply:
So it does! Turns out I was looking at the wrong thing... nothing to see here!